release/ray_release/tests/test_test.py

```diff
@@ -140,6 +140,23 @@ def test_get_ray_image():
         )
 
 
+def test_get_byod_runtime_env():
+    test = _stub_test(
+        {
+            "python": "3.11",
+            "cluster": {
+                "byod": {
+                    "runtime_env": ["a=b"],
+                },
+            },
+        }
+    )
+    runtime_env = test.get_byod_runtime_env()
+    assert runtime_env.get("RAY_USAGE_STATS_SOURCE") == "nightly-tests"
+    assert runtime_env.get("RAY_BACKEND_LOG_JSON") == "1"
+    assert runtime_env.get("RAY_USAGE_STATS_EXTRA_TAGS") == "test_name=test"
```

Comment:
should somewhere here tests `a=b`? the one inherited from the test?

Reply:
ah yes added